src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/Providers/IaasVmPsBackupProvider.cs

```diff
@@ -77,5 +78,113 @@ public void DeletePolicy()
         {
             throw new NotImplementedException();
         }
+
+
+        public AzureRmRecoveryServicesSchedulePolicyBase GetDefaultSchedulePolicyObject()
+        {
+            AzureRmRecoveryServicesSimpleSchedulePolicy defaultSchedule = new AzureRmRecoveryServicesSimpleSchedulePolicy();
+            //Default is daily scedule at 10:30 AM local time
+            defaultSchedule.ScheduleRunFrequency = ScheduleRunType.Daily;
+
+            DateTime scheduleTime = GenerateRandomTime();
+            defaultSchedule.ScheduleRunTimes = new List<DateTime>();
+            defaultSchedule.ScheduleRunTimes.Add(scheduleTime);
+
+            defaultSchedule.ScheduleRunDays = new List<DayOfWeek>();
+            defaultSchedule.ScheduleRunDays.Add(DayOfWeek.Sunday);
+
+            //TimeSpan triggerTime = new TimeSpan(10, 30, 00);
+            //TimeSpan timeToGo = triggerTime - DateTime.Now.TimeOfDay;
+            //if(timeToGo < TimeSpan.Zero)
+            //{
+            //    //Schedule time will be the next day
+            //}
+            //else
+            //{
+            //    //Schedule time will be today 10:30
+            //}
+
+            return defaultSchedule;
+        }
+
+        public AzureRmRecoveryServicesLongTermRetentionPolicy GetDefaultRetentionPolicyObject()
+        {
+            AzureRmRecoveryServicesLongTermRetentionPolicy defaultRetention = new AzureRmRecoveryServicesLongTermRetentionPolicy();
+            
+            //Default time is 10:30 local time
+            DateTime retentionTime = GenerateRandomTime(); 
```

Comment:
This retention time should be same as of BackupTime. Use that time instead of generating it one more time,